plutus-core/plutus-core/src/PlutusCore/Annotation.hs

```diff
@@ -76,6 +76,8 @@ data SrcSpan = SrcSpan
     , srcSpanSCol  :: Int
     , srcSpanELine :: Int
     , srcSpanECol  :: Int
+    -- ^ The ending column is one more than the column of the last character (same as
```

Comment:
What happens if the last character is the last token in the line? Does that mean that this will be the index of a non-existent character?

Reply:
Normally there's at least a line break character following the last character.
But technically the last character could be the line break, so I'll update the comment

Reply:
Updated the comment.